integration_tests/ethereum/contracts/MockAaveV2StablecoinCellar.sol

```diff
@@ -1,57 +1,72 @@
 // SPDX-License-Identifier: Apache-2.0
 pragma solidity ^0.8.0;
 
-import "./interfaces.sol";
+import {ERC20, Ownable} from "./interfaces.sol";
 
 contract MockAaveV2StablecoinCellar is Ownable {
     /**
      * @notice Whether or not the contract is shutdown in case of an emergency.
      */
     bool public isShutdown;
 
-    event mockAccrueFees();
-    event mockTransferFees();
+    event mockAccrue();
+    event mockSendFees();
     event mockSetFeesDistributor(bytes32 newFeesDistributor);
-    event mockEnterPosition();
+    event mockEnterPosition(uint256 assets);
```

Comment:
I believe the mock contract is still missing the enter/exit aliases with no arguments.